projects/storefrontlib/cms-components/product/product-list/product-facet-navigation/active-facets/active-facets.component.html

```diff
@@ -8,11 +8,6 @@ <h4 *ngIf="facetList?.activeFacets?.length > 0">
     routerLink="./"
     [queryParams]="getLinkParams(facet)"
     [cxFocus]="getFocusKey(facetList, facet)"
-    role="button"
```

Comment:
Why change this file? This doesn't seem to fit the scope of the PR.
Let's discuss.

Reply:
Don't know why this is here. I didn't make any changes to this file. Maybe during a /develop branch merge it changed these files

Reply:
In an earlier commit when merging a branch to this one, these changes existed. The revert of that commit is undoing those changes.